sci-libs/vtk/vtk-9.0.1.ebuild

```diff
@@ -0,0 +1,546 @@
+	https://www.vtk.org/files/release/${SPV}/VTKData-${PV}.tar.gz
+	doc? ( https://www.vtk.org/files/release/${SPV}/vtkDocHtml-${PV}.tar.gz )
+	examples? ( https://www.vtk.org/files/release/${SPV}/VTKLargeData-${PV}.tar.gz )
+	test? (
+		https://www.vtk.org/files/ExternalData/MD5/eaab2cdba68e8630f9a1f8a8e81cb097
+		https://www.vtk.org/files/ExternalData/MD5/325acbb1a74ae27148d76399c2e7c7c5
+		https://www.vtk.org/files/ExternalData/MD5/8e09d028a9fd114b4ceb8f924d8b466c
+		https://www.vtk.org/files/ExternalData/MD5/7e7e6c5e28fa93b16fe0414881bdbbcd
+		https://www.vtk.org/files/release/${SPV}/VTKLargeData-${PV}.tar.gz
+	)
+"
+
+LICENSE="BSD LGPL-2"
+# several dependencies need to be keyworded for arm
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+SLOT="0"
+# New in 9.0?: sdl -> from which group / module is it coming?
+# Note: external xdmf2 has no recognized targets
+IUSE="+X all-modules boost cuda doc +eigen examples ffmpeg gdal imaging
+	java +json kits mpi mysql odbc offscreen openmp pegtl postgres python
+	qt5 +rendering tbb theora tk video_cards_nvidia views web"
+
+# most tests don't work
+RESTRICT="test"
+
+REQUIRED_USE="
+	all-modules? ( boost ffmpeg gdal mysql odbc postgres theora )
+	cuda? ( X )
+	python? ( ${PYTHON_REQUIRED_USE} )
+	qt5? ( X rendering )
+	tk? ( X rendering python )
+	web? ( python )
+"
+
+RDEPEND="
+	app-arch/lz4
+	app-arch/xz-utils
+	dev-db/sqlite
+	dev-cpp/eigen[cuda?,openmp?]
+	dev-libs/double-conversion:=
+	dev-libs/expat
+	dev-libs/icu:=
+	dev-libs/libxml2:2
+	dev-libs/pugixml
+	dev-libs/utfcpp
+	media-libs/libogg
+	media-libs/libpng
+	media-libs/libtheora
+	media-libs/tiff
+	sci-libs/hdf5:=[mpi=]
+	sci-libs/netcdf:=[mpi=]
+	sys-libs/zlib
+	virtual/jpeg
+	all-modules? ( sci-geosciences/liblas[gdal] )
+	boost? ( dev-libs/boost:=[mpi?] )
+	cuda? ( dev-util/nvidia-cuda-toolkit:= )
+	ffmpeg? ( media-video/ffmpeg:= )
+	gdal? ( sci-libs/gdal:= )
+	java? ( >=virtual/jdk-1.8:* )
+	json? ( dev-libs/jsoncpp:= )
+	mpi? (
+		sci-libs/h5part
+		sys-cluster/openmpi[cxx,romio]
+		virtual/mpi[cxx,romio]
+	)
+	mysql? ( virtual/mysql )
+	odbc? ( dev-db/unixODBC )
+	offscreen? ( media-libs/mesa[osmesa] )
+	pegtl? ( dev-libs/pegtl )
+	postgres? ( dev-db/postgresql:= )
+	python? ( ${PYTHON_DEPS} )
+	qt5? (
+		dev-qt/qtcore:5
+		dev-qt/qtsql:5
+		dev-qt/qtwidgets:5
+	)
+	rendering? (
+		media-libs/freeglut
+		media-libs/glew:=
+		sci-libs/proj:=
+		virtual/opengl
+		x11-libs/gl2ps
+	)
+	tbb? ( dev-cpp/tbb )
+	tk? ( dev-lang/tk:= )
+	video_cards_nvidia? ( x11-drivers/nvidia-drivers[tools,static-libs] )
+	views? (
+		x11-libs/libICE
+		x11-libs/libXext
+	)
+	web? ( ${WEBAPP_DEPEND} )
+	$(python_gen_cond_dep '
+		python? (
+			all-modules? ( dev-python/mpi4py[${PYTHON_MULTI_USEDEP}] )
+			boost? ( dev-libs/boost:=[mpi?,python?,${PYTHON_MULTI_USEDEP}] )
+			gdal? ( sci-libs/gdal:=[python?,${PYTHON_MULTI_USEDEP}] )
+			mpi? ( dev-python/mpi4py[${PYTHON_MULTI_USEDEP}] )
+		)
+	')
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	dev-libs/jsoncpp:=
```

Comment:
:= [doesn't do anything](https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-820008.3.3) as a BDEPEND, fwiw. Also, does jsoncpp provide a binary or something we call?

Reply:
No binary that get's called, but it's not linked into the final binaries or libraries AFAIK. Probably needed by some helper binaries. So it goes into DEPEND I suspect?